chart/values.yaml

```diff
@@ -4,7 +4,6 @@ version: 0.1.9
 
 image:
   name: oamdev/terraform-controller:latest
-  registry: docker.io/daprio
   imagePullPolicy: Always
 
-namespace: default
\ No newline at end of file
+velaNamespace: vela-system
```

Comment:
```suggestion
namespace: vela-system
```

This controller can work independently without KubeVela, so shall we keep the orignal key?

Reply:
Good point; this variable is used by the `tf-controller-read-provider-creds-rolebinding` to allow the controller to read the cloud provider credentials secret, so perhaps `credentialsNamespace` or `credentialsSecretNamespace`?

Reply:
`credentialsNamespace` +1.

Reply:
Changed in f7bb5c22d1219f3886780708d015d1074c78476d